config/v1/types_console.go

```diff
@@ -20,11 +19,12 @@ type Console struct {
 }
 
 type ConsoleSpec struct {
-	// special console public url?
+	Auth Auth `json:"auth"`
```

Comment:
Does this imply authentication, authorization or both.

Reply:
I don't think we will have enough fields on this struct to separate them, so probably both.  At present, we plan on exactly 1 field (see below).

Reply:
Update, will change to `Authentication`.  Can add `Authorization` later if needed.